app/bundles/LeadBundle/EventListener/ReportSubscriber.php

```diff
@@ -815,6 +825,30 @@ private function injectFrequencyReportData(ReportBuilderEvent $event, array $col
      * @param ReportBuilderEvent $event
      * @param array              $columns
      */
+    private function injectSegmentReportData(ReportBuilderEvent $event, array $columns)
+    {
+        $segmentColumns = [
+            'lll.manually_removed' => [
+                'label' => 'mautic.lead.report.segment.manually_removed',
+                'type'  => 'bool',
+            ],
+            'lll.manually_added' => [
+                'label' => 'mautic.lead.report.segment.manually_added',
+                'type'  => 'bool',
+            ],
+        ];
+
+        $data = [
+            'display_name' => 'mautic.lead.report.segment.membership',
+            'columns'      => array_merge($columns, $segmentColumns, $event->getIpColumn()),
```

Comment:
Merge in `$event->getStandardColumns('ll')` here so that segment name, etc are available to the columns.